shared/test/test_assets.rb

```diff
@@ -182,6 +183,40 @@ def test_viewing_abusive_assets
     @api.delete_object(asset_name)
   end
 
+  def test_assets_copy_some
+    delete_all_assets('assets_test/1/2')
+    dest_channel_id = create_channel
+    src_api = FilesApiTestHelper.new(current_session, 'assets', @channel_id)
+    dest_api = FilesApiTestHelper.new(current_session, 'assets', dest_channel_id)
+
+    image_filename = 'çat.jpg'
+    image_body = 'stub-image-contents'
+
+    sound_filename = 'woof.mp3'
+    sound_body = 'stub-sound-contents'
+
+    _, image_filename = post_asset_file(src_api, image_filename, image_body, 'image/jpeg')
+    _, sound_filename = post_asset_file(src_api, sound_filename, sound_body, 'audio/mpeg')
+    src_api.patch_abuse(10)
```

Comment:
What's the desired behavior with respect to projects with an abuse score, should it be possible to export them?

It looks like you are not verifying that the abuse score is reset to zero as we are doing in `test_assets_copy_all`. I would recommend removing `src_api.patch_abuse(10)` here, or else putting back the checks which confirm the score is reset (if that's the desired behavior).

@Bjvanminnen can you explain why we are clearing the abuse score when we copy images on remix?